components/webpack/webpack.config.js

```diff
@@ -89,8 +89,19 @@ module.exports = async function (env, argv) {
       concatenateModules: !process.env.NO_CONCATENATE
     },
     experiments: {
-      syncWebAssembly: true
+      syncWebAssembly: true,
+      outputModule: Boolean(env.output_module)
     },
+    externals: [
+      function ({ context, request }, callback) {
+        if (env.output_module) {
+          if (/^chrome:\/\//.test(request)) {
```

Comment:
We need to match also `chrome-untrusted://`. I'm wondering if we need to match `//` which is `chrome`/`chrome-untrusted`-agnostic, but perhaps we don't need to worry about that as the top layer of component-specific TS will know which one to use, and the common files which may use `//` would not be bundled by webpack?

Reply:
I think we probably do need to worry about `//` because we need to mark some (all?) of them as external right?

Reply:
With this change, protocol-relative will already be treated as external, but if we use it in our own TS code, TypeScript will error out (since it doesn't know where to look currently). I was thinking we could add TS support for `//` with a follow-up?

Reply:
I'm going to experiment for a bit with `//` though...

Reply:
I've confirmed that webpack will currently treat `//` as external with this change, and so the only barrier to using it in our own code is getting TS to understand where to look. @petemill are you OK with making the TS config change to support `//` imports in a follow up?

Reply:
But this change only affects webpack not typescript right? So as long as the first external found is chrome:// or chrome-untrusted:// and the protocol-relative path is inside those files, webpack won't see them as it won't look inside those external files...?